front/src/components/search/SearchContainer.tsx

```diff
@@ -22,13 +33,28 @@ const EmptyListMessage = () => (
   </Flex>
 );
 
-const WelcomeMessage = () => (
+const WelcomeMessage = ({ colorMode }: Pick<Props, 'colorMode'>) => (
   <Flex justifyContent="center">
-    <Text>Welcome, search for a show</Text>
+    <StatGroup
+      bg={colorMode === 'light' ? 'white' : ''}
+      d="flex"
+      alignItems="center"
+      flexDirection={{ sm: 'column', lg: 'row' }}
+      borderWidth="1px"
+      padding="15px"
+      shadow="sm"
+      minHeight="125px"
+    >
+      <Stat d="flex" flexDirection="column" alignItems="center">
+        <StatLabel>Keep track of your favorite shows with this app</StatLabel>
+        <StatHelpText>Search for a show now !</StatHelpText>
```

Comment:
Let's delete the space between now and !

Reply:
will do